dotnet/samples/PlayFabExamples/Example02_Generative/Example02_Generative.cs

```diff
@@ -42,7 +41,7 @@ private static async Task CreateSegmentExample(string goal)
         Console.WriteLine("======== Action Planner ========");
         var kernel = new KernelBuilder()
             .WithLogger(ConsoleLogger.Logger)
-            .WithAzureTextCompletionService(TestConfiguration.AzureOpenAITextDavinci.DeploymentName, TestConfiguration.AzureOpenAITextDavinci.Endpoint, TestConfiguration.AzureOpenAITextDavinci.ApiKey)
+            .WithAzureTextCompletionService(TestConfiguration.AzureOpenAI.DeploymentName, TestConfiguration.AzureOpenAI.Endpoint, TestConfiguration.AzureOpenAI.ApiKey)
```

Comment:
TextDavinci deployment name should be different.

Reply:
For each model, do we want to keep same end point and key? In the idea environment, we can keep the same.

Reply:
For keeping the configuration simple yes, we can assume all models are on the same endpoint (and hence using same key)